jenkins/agent-ctl.sh

```diff
@@ -92,28 +103,43 @@ fi
 case "${KIND}" in
   light)
     # Current experiment:
-    # 20 executors, n1-highmem-16, 500G pd-standard
+    # 14 agents
+    # 10 executors, n1-highmem-8, 250G pd-standard
     # Results:
-    # 5.31 cores, 70G ram, <500 write IOPs, <50MB/s write
-    # Alernate experiment:
-    # 2 executors, n1-highmem-2, 100G pd-standard
+    # (1.0 cores, 20G active ram)
+    # 1.46 cores, 50G ram, <250 write IOPs, <25MB/s write
     # Results:
     # 0.2 cores, 1.3G ram, low IOPs (200 IOP spikes), low write (30MB/s spikes)
-    DISK_SIZE='500GB'
+    DISK_SIZE='250GB'
     DISK_TYPE='pd-standard'
-    MACHINE_TYPE='n1-highmem-16'
+    MACHINE_TYPE='n1-highmem-8'
     ;;
   heavy)
-    # TODO(fejta): optimize these values
-    DISK_SIZE='200GB'
+    # Current experiment:
+    # 8 agents
+    # 1 executor, n1-highmem-8, 150G pd-standard
+    # Results:
+    # 14-32 cores, 12G ram, 150 write IOPs, <20MB/s write
+    DISK_SIZE='150GB'
     DISK_TYPE='pd-standard'
-    MACHINE_TYPE='n1-standard-16'
+    MACHINE_TYPE='n1-standard-8'
```

Comment:
If we only have a handful of heavy post-submit agents I think we can switch to n1-standard-32.